_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,289 @@
+
+You can check that a Pod is created for our application by running:
+
+```bash
+kubectl get pods
+```
+
+which should return a similar output to:
+
+```bash
+NAME                                       READY     STATUS    RESTARTS   AGE
+laravel-kubernetes-demo-7dbb9d6b48-q54wp   1/1       Running   0          18m
+```
+
+You can also use the Minikube GUI dashboard to monitor the cluster. The GUI also helps with visualising most of the discussed concepts. To view the dashboard, just run the following:
+
+```bash
+minikube dashboard
+```
+
+or to acquire the dashboard's URL address:
+
+```bash
+minikube dashboard --url=true
+```
+
+__Exposing a service__
+
+So far you have created a deployment which is running the application's container. A pod running in the cluster has a dynamic IP. If you route the traffic directly to it using the IP, you may still need to update the routing table every time the pod is restarted. In fact, on every deployment or container restart, a new IP is assigned to the pod. To avoid managing IP addresses manually, you need to use a Service. The service acts as an IP and serves as a load balancer for a set of pods. So even if the IP address of a pod changes, the service is always pointing to it. And since the Service always has the same IP, you won't need to update anything manually.
+
+```bash
+kubectl expose deployment laravel-kubernetes-demo --type=NodePort --port=8181
+```
+
+and provided all went well, you will see a confirmation similar to:
+
+```bash
+service "laravel-kubernetes-demo" exposed
+```
+Running the following command:
+
+```bash
+kubectl get services
+```
+will show you a list of running services. You can also view the running service under the "Services" navigation menu within the dashboard. A more exciting way to verify this deployment and the service exposure is obviously seeing the running application in the browser 😊
+
+To obtain the URL of the application (service), you can use the following command:
+
+```bash
+minikube service --url=true laravel-kubernetes-demo
+```
+
+which will output the IP address and port number similar to:
+
+```bash
+http://192.168.99.101:31399
+```
+
+or, launch the application directly in the browser:
+
+```bash
+minikube service laravel-kubernetes-demo
+```
+
+__Scaling__
+
+And that is it. You have successfully deployed the application in Kubernetes. It's exciting. But what's the point of doing all of this? Well, you only have one deployment with a single Pod running, provisioned to a Node with the exposed web service. Let's replicate this pod to two more pods acting as load balancers to the web traffic.
```

Comment:
Let's replicate this pod to two more pods acting as load balancers to the web traffic.

Not sure what you meant with "acting as load balancers to the web traffic".

I think you could have: "Let's scale this deployment to two instances of the application"

Also note that later on you scale your replicas to 3. Not sure which one you wish to keep.

Reply:
I agree with the text change. The numbers are correct though :) 1 + 2 __more__ = three 😄

Reply:
OH LOL. SORRY.